src/dotnet/commands/dotnet-msbuild/MSBuildLogger.cs

```diff
@@ -17,36 +16,58 @@ public sealed class MSBuildLogger : Logger
 
         public MSBuildLogger()
         {
-            string sessionId = Environment.GetEnvironmentVariable(MSBuildForwardingApp.TelemetrySessionIdEnvironmentVariableName);
+            try
+            {
+                string sessionId = Environment.GetEnvironmentVariable(MSBuildForwardingApp.TelemetrySessionIdEnvironmentVariableName);
```

Comment:
NIT: break the line here. Keep it at less than 120 characters. Horizontal scrolling is very inconvenient.